src/clang_complete.cc

```diff
@@ -776,7 +776,7 @@ std::shared_ptr<CompletionSession> ClangCompleteManager::TryGetSession(
     // If this request is for a completion, we should move it to
     // |completion_sessions|.
     if (mark_as_completion) {
-      assert(!completion_sessions_.TryGet(filename));
+      // assert(!completion_sessions_.TryGet(filename));
```

Comment:
Related to https://github.com/MaskRay/ccls/issues/10 ?

Reply:
Yeah. Seems to be not harmful.